spec/models/user_adds_review_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+RSpec.describe ReviewAdded, :type => :model do
+
+  describe "valid review" do
```

Comment:
this seems to be a bit confused between being a unit test and an acceptance test. You want your unit tests to exercise individual instance methods of your model